deploy/kubernetes/csi-cvmfsplugin.yaml

```diff
@@ -62,19 +62,22 @@ spec:
               readOnly: true
             - name: host-dev
               mountPath: /dev
+            - name: cvmfs-config
```

Comment:
If you feel like it, it would be nice to move this to a separate PR. Otherwise leave it, that's ok too.

Reply:
I agree it is not strictly needed for moving to v1.